### DISCUSSION TITLE
R14 Gear Launch Suggestion

### DISCUSSION THREAD

**Discussion by Scrublettt** [chromiecraft/chromiecraft#3734] [Suggestions]
Hi all! Yesterday we've had a nice discussion in the pvp channel on Discord about what we think R14 Gear requirements should look like.

After that I have drafted a proposal, based on the discussion, which so far has a positive feedback.

Here it is:

**1. 3k HK requirement - unlocks R14 set pieces. Before this, you can buy Blue PVP gear.**

- it is equivalent to getting Honored rep with all BG Forces
- Play pvp and you get it anyway (most players who occasionally play PVP should have it already)
- by the time you get this, you should have farmed enough honor to get Blue PVP set
- Gates fresh characters from getting R14 gear instantly - you get Blue set first and when you reach 3k HKs, you go for R14
- Alt friendly 

**2. BG Related Achievement Requirement - unlocks R14 weapons**

Here we need Dev opinion if its possible and how hard it will be to implement. The options that we came with are:

2.1. Have a set of achievements

WSG - Not So Fast, Warsong Experience, Warsong Gulch Perfection, Capture the Flag
AB - Me and the Cappin' Makin' it Happen, Arathi Basin All-Star, Let's Get This Done, Disgracin' The Basin
AV - The Stickly Gazelle, Alterac Valley Victory, Alterac Valley All-Star

- All the achievements above are not hard to obtain. Most of them people get without trying.
- Having achievement requirement for all 3 BGs should make people que more frequently for AB and AV 
- By going for those achievements you are actually helping your team winning 
- The WSG achievements can be done in a single game
- To get the AB achievements may take something like 10-15 games. This is intentional, so we make ppl que for AB more frequently (everyone loves AB more than WSG).
- The AV achievements are again easy to obtain in a single game. The main requirement here is to get a win and do a few objectives for the All-Star achievement (which again helps your team winning).
*based on AV popularity, AV CTA may be required for a player to actually have a chance on getting the achievement.

2.2. 200 of PVP (or BG) related Achievement Points

- Each achievement brings an amount of achievement points. Some give 10, some give more.
- This way it wont matter what combination of achievements you got, as long as they are PVP related.
- This way we actually make people to focus on getting the achievements and not only HK farming.

**3. Gear cost** 

3.1. Set piece - 50,000 honor + 10 WSG marks + 5 AB marks
3.2. Weapon - 75,000 honor + 20 WSG marks + 10 AB marks + 3 AV marks

- Reason here is to make people actually que different BGs to get the marks they are missing and not only spam the current CTA to get max honor.
- Traditional PVP players wont be getting full gear right away. They will need some investment too.

I believe that this system is new player / alt friendly enough, should get more PVE players on the PVP scene, as its not a brainless grind, and still gives a slight advantage for seasoned PVP players.

**Comment by Vitusin** [comment]
It was hard fight, but Yeah thanks for this suggestion. Lets thumbs up it to the moon. Best idea for r14 yet.

**Comment by plasma-93** [comment]
Well reasoned, fair, and most importantly, fun. I love the idea of mixing in achievements, this makes it a lot more fun for casuals, and adds ab it of a 'minigame' vibe to getting the requirements.

1. Like this, 3K is a good number imo

2. Would rather x amount of AB / AV / WSG achievement points as opposed to specific achievements, but both are cool.

3. Only don't like the WSG marks because WSG sucks :)


I think the best part about this suggestion is that it actually makes the grind engaging, rather than droning through games for many many hours, you have a variety of objectives which makes it approachable. Emergent stuff like "alright boys I know I'm a fury warrior but I need to cap the flag in 75sec" is some of the best content around lol

**Comment by RestoCC** [comment]
Well-written and thought out. Very much in favour of this even as someone that doesnt do much PvP. Upvoted.

**Comment by balleny** [comment]
for the sake of making it less complex to implement, my previous suggestion:
https://github.com/chromiecraft/chromiecraft/discussions/3457

gatekeeping the items based on the amount of honored kills paired with a rather high amount of honor per item. People, who pvp'ed a lot, wil have earlier access (as it was the case in retail) and everyone will get access at a later point (as it was in retail with 2.0.1 prepatch)

**Comment by djobf** [comment]
100% agreed, well thought out and put together

**Comment by DarylBlake21** [comment]
Tbh it's good to see BG marks having another use cause after you finish the rep grind at exalted they're pretty useless.

**Reply by balleny** [reply]
because they arent be supposed to be obtainable in wrath.

**Comment by meudeusdosceus** [comment]
Its a lot of work, people will need to spend some time, but sounds fun and fair.

**Comment by ishidinho** [comment]
First of all, thank you!

Your suggestions addresses my main problem with the current state of getting R14 gear very well. This problem is that the HK farming players stand in the players ways that want to farm honor, reputation or just want to play the actual battleground. We need to find a solution that
1. let people play the battlegrounds how they are supposed to be played ,
2. where the players are noch standing in other players way on their way to R14 gear, honor, reputation or PvP fun in general and
3. every player can make continous progress on his goal.

Regarding point (2) I would remove the achievement requirements where you have to win the battleground in a short amount of time.
For example, a player who needs to cap 50 bases in AB is not interested in winning within 6 minutes, which another player in his group wants to do.
